Autopilot/Src/MPXV7002DP.cpp

```diff
@@ -0,0 +1,92 @@
+#include "airspeed.hpp"
+#include "adc.h"
+#include <stddef.h>
+
+#include <cmath>
+
+#define VS 5
+#define DENSITY_AIR 1.2754
+#define ADC_RESOLUTION 4096
+#define KPA_TO_PA 1000
```

Comment:
PA_TO_KPA (KPA_TO_PA is 0.001)

Reply:
I think I might be confused. 1 KPa = 1000 Pa, so to go from KPa to Pa it should be Kpa * 1000, right?

Reply:
I am stupid. Please ignore this